libs/features/georoutingv2/georoutingv2.js

```diff
@@ -257,7 +257,7 @@ async function showModal(details) {
   const promises = [
     tabs ? loadBlock(tabs) : null,
     tabs ? loadStyle(`${miloLibs || codeRoot}/blocks/section-metadata/section-metadata.css`) : null,
-    loadStyle(`${miloLibs || codeRoot}/features/georoutingv2/georoutingv2.css`),
+    new Promise((resolve) => { loadStyle(`${miloLibs || codeRoot}/features/georoutingv2/georoutingv2.css`, resolve); }),
```

Comment:
we'd need to do the same for the `loadStyle` above, right?

Reply:
Hi Okan, I've created a refactor ticket to look into this. [MWPW-152228](https://jira.corp.adobe.com/browse/MWPW-152228)